test/Sentry.Tests/Protocol/TransactionTests.cs

```diff
@@ -504,4 +504,17 @@ public void ISpan_GetTransaction_FromSpan()
         // Assert
         Assert.Same(transaction, result);
     }
+
+    [Fact]
+    public async Task Idle_transaction_should_finish_with_idle_timeout_specified()
+    {
+        // Arrange
+        var transaction = new TransactionTracer(DisabledHub.Instance, "my name", "my op", TransactionNameSource.Custom, TimeSpan.FromMilliseconds(10));
```

Comment:
@bitsandfoxes, I chose a 10 milliseconds timeout and ensured it's invoked with a delay of 15 ms.
Thinking about it, the idle timeout and the delay could have been even less than that. On the flip side, this is not a significantly substantial amount of time. What do you think, keep it as-is or reduce it?

Reply:
Changed to lower values.